libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/utils/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.navigation.base.internal.utils;
+
+public final class Constants {
+
+    public static final class RouteResponse {
+        public static final String KEY_WAYPOINTS = "waypoints";
+    }
+}
```

Comment:
Because in kotlin this code:
```
object ConstantsKt {
    object RouteResponse {
        const val KEY_WAYPOINTS = "waypoints"
    }
}
```
looks like this:
```
public final class ConstantsKt {
   @NotNull
   public static final ConstantsKt INSTANCE;

   private ConstantsKt() {
   }

   static {
      ConstantsKt var0 = new ConstantsKt();
      INSTANCE = var0;
   }

   @Metadata(
      mv = {1, 5, 1},
      k = 1,
      d1 = {"\u0000\u0012\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\b\u0002\n\u0002\u0010\u000e\n\u0000\bÆ\u0002\u0018\u00002\u00020\u0001B\u0007\b\u0002¢\u0006\u0002\u0010\u0002R\u000e\u0010\u0003\u001a\u00020\u0004X\u0086T¢\u0006\u0002\n\u0000¨\u0006\u0005"},
      d2 = {"Lcom/mapbox/navigation/base/internal/utils/ConstantsKt$RouteResponse;", "", "()V", "KEY_WAYPOINTS", "", "mapbox-navigation-android.libnavigation-base.main"}
   )
   public static final class RouteResponse {
      @NotNull
      public static final String KEY_WAYPOINTS = "waypoints";
      @NotNull
      public static final ConstantsKt.RouteResponse INSTANCE;

      private RouteResponse() {
      }

      static {
         ConstantsKt.RouteResponse var0 = new ConstantsKt.RouteResponse();
         INSTANCE = var0;
      }
   }
}
```
Isn't it too much for a string constant?
I can add the `JvmStatic` annotation, but it's not much better.
Do you have anything against java? I can easily convert it to kotlin but I think that for the purposes like this (when you just need a holder for some constants) java is better (probably the only situation where I'd say this).

Reply:
Converted to kotlin.

Reply:
Thanks, that's an interesting point of view. I guess it's a general problem with `object`s that they are just singletons under the hood after compilation. Either way would work, I was just curious what was the motivation.